admin_manual/configuration_database/linux_database_configuration.rst

```diff
@@ -20,36 +20,19 @@ requires that you install and set up the server software first.
   scope of this document.  Please refer to the documentation for your specific
   database choice for instructions.
 
-.. _db-transaction-label:
-
-Database "READ COMMITTED" transaction isolation level
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-As discussed above Nextcloud is using the ``TRANSACTION_READ_COMMITTED`` transaction isolation
```

Comment:
Hi, can we keep this section to have the transaction isolation level documented somewhere? I guess the rest can go.